pkg/deploy/strategy/support/lifecycle.go

```diff
@@ -67,11 +74,24 @@ func NewHookExecutor(client kclient.PodsNamespacer, tags client.ImageStreamTagsN
 // Execute executes hook in the context of deployment. The suffix is used to
 // distinguish the kind of hook (e.g. pre, post).
 func (e *HookExecutor) Execute(hook *deployapi.LifecycleHook, deployment *kapi.ReplicationController, suffix, label string) error {
-	var err error
+	config, err := deployutil.DecodeDeploymentConfig(deployment, e.decoder)
+	if err != nil {
+		return err
+	}
+
 	switch {
 	case len(hook.TagImages) > 0:
+		tagEventMessages := []string{}
+		for _, t := range hook.TagImages {
+			image, ok := findContainerImage(deployment, t.ContainerName)
+			if ok {
+				tagEventMessages = append(tagEventMessages, fmt.Sprintf("image %s to %s", image, t.To.Name))
+			}
+		}
+		e.recorder.Eventf(config, kapi.EventTypeNormal, "Tagged", "Tagging %s", strings.Join(tagEventMessages, ","))
 		err = e.tagImages(hook, deployment, suffix, label)
 	case hook.ExecNewPod != nil:
+		e.recorder.Eventf(config, kapi.EventTypeNormal, "Executed", "Running %s-hook command %q in container %q", suffix, strings.Join(hook.ExecNewPod.Command, " "), hook.ExecNewPod.ContainerName)
```

Comment:
I don't think we need to be that verbose here. How about `Started deployment post-hook for {name-of-deployment}`?